interface-definitions/service_monitoring_prometheus.xml.in

```diff
@@ -36,6 +36,137 @@
                   #include <include/interface/vrf.xml.i>
                 </children>
               </node>
+              <node name="blackbox-exporter">
+                <properties>
+                  <help>Prometheus exporter for probing endpoints</help>
+                </properties>
+                <children>
+                  #include <include/listen-address.xml.i>
+                  #include <include/port-number.xml.i>
+                  <leafNode name="port">
+                    <defaultValue>9115</defaultValue>
+                  </leafNode>
+                  #include <include/interface/vrf.xml.i>
+                  <node name="modules">
+                    <properties>
+                      <help>Configure blackbox exporter modules</help>
+                    </properties>
+                    <children>
+                      <node name="dns">
+                        <properties>
+                          <help>Configure dns module</help>
+                        </properties>
+                        <children>
+                          <tagNode name="name">
+                            <properties>
+                              <help>Name of the dns module</help>
+                            </properties>
+                            <children>
+                              <leafNode name="query-type">
+                                <properties>
+                                  <help>DNS query type</help>
+                                  <constraint>
```

Comment:
Please add valueHelp XML definitions for A, AAAA and ANY - Can I also test TXT or NS?

Reply:
Any kind of DNS record can be requested